setuptools/tests/test_egg_info.py

```diff
@@ -148,6 +148,26 @@ def test_expected_files_produced(self, tmpdir_cwd, env):
         ]
         assert sorted(actual) == expected
 
+    def test_license_is_a_string(self, tmpdir_cwd, env):
+        setup_config = DALS("""
+            [metadata]
+            name=foo
+            version=0.0.1
+            license=file:MIT
+            """)
+
+        setup_script = DALS("""
+            from setuptools import setup
+
+            setup()
+            """)
+
+        build_files({'setup.py': setup_script,
+                     'setup.cfg': setup_config})
+
+        with pytest.raises(AssertionError) as exception:
```

Comment:
```suggestion
        with pytest.raises(ValueError):
```

I'm a bit distured that this was succeeding in the first place, as this *should* be raising `ValueError`.